webapp/components/Editor/Editor.vue

```diff
@@ -205,10 +205,13 @@ export default {
       if (!query) {
         return items
       }
-      return items.filter(item => {
+
+      items = this.suggestionType === HASHTAG ? this.hashtags : this.users
```

Comment:
I think I would rather pass all items (hashtags or users) to the new `Hashtag` or `Mention` instead of checking for the suggestion type again during the filtering process and then resetting items to the corresponding type.

We could still limit the number of displayed suggestions by returning `items.slice(0, 15)` when there is no query (in line `206`).

Seems a bit cleaner to me. What do you think?

Reply:
yes, it does! I got it working, but then didn't take a step back to think about it critically... good catch!